components/dropzone/index.css

```diff
@@ -56,7 +56,8 @@ governing permissions and limitations under the License.
   --spectrum-drop-zone-content-color: var(--spectrum-white);
 
   /* Settings for a nested illustrated message */
-  --mod-illustrated-message-pointer-events: none;
+  --mod-illustrated-message-pointer-events: initial;
+  --mod-illustrated-message-description-pointer-events: initial;
```

Comment:
I think we could remove these two lines entirely and get the same computed value of `auto` (its initial value) on `pointer-events` in Illustrated Message, since it is only setting it to the mod like this `pointer-events: var(--mod-illustrated-message-pointer-events);`

Reply:
Good callout. I updated that in my second commit and then set defaults in Illustrated Message [here](https://github.com/adobe/spectrum-css/pull/2301/commits/a0c4c626e54daeed5b39442fafcb36ae0269893e).